crates/tbdex/src/json_schemas/generated.rs

```diff
@@ -241,7 +241,7 @@ pub const OFFERING_DATA_JSON_SCHEMA: &str = r#"{
           "type": "boolean",
           "description": "Whether cancellation is enabled for this offering"
         },
-        "terms_url": {
+        "termsUrl": {
```

Comment:
this was expected right?

Reply:
correct insofar as I didn't update the json schema hash in the target merge branch here 

but yeah it's fine